app/controllers/iframes.js

```diff
@@ -1,12 +1,11 @@
 import Ember from "ember";
-var ArrayController = Ember.ArrayController;
-var alias = Ember.computed.alias;
-var mapComputed = Ember.computed.map;
-var run = Ember.run;
+const { ArrayController, computed, run } = Ember;
+const { alias, map } = computed;
+const mapComputed = map;
```

Comment:
This should either be `const { alias, map: mapComputed } = computed` or we just use `map` in the file below.